server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/pullrequest/review/PullRequestReviewState.java

```diff
@@ -0,0 +1,5 @@
+package de.tum.in.www1.hephaestus.codereview.pullrequest.review;
+
+public enum PullRequestReviewState {
+    COMMENTED, APPROVED, CHANGES_REQUESTED, DISMISSED
```

Comment:
is OPEN as state missing? or does it not exist?

Reply:
I think it does not exist for a review

Reply:
exactly, it's 1:1 converted from this enum: https://github-api.kohsuke.org/apidocs/org/kohsuke/github/GHPullRequestReviewState.html